### ISSUE TITLE
About training epochs and learning rate

### ISSUE THREAD

**Issue by jingchunhui** [Jiahao-UTS/SLPT-master#9]
Hi, what a great work!
But i still have some questions in training the model.
The initial learning rate is 0.001 in your paper and I really wonder how the learning rate changes and  how many epochs you have trained for this model?
These will be very usefully for me to learn more about your paper, thank you very much!

**Comment by tomaleg** [comment]
Hi,
If you can look in the appendix section, in the "A.3 Convergence curves of SLPT and DETR" you can find that the SLPT seems to be train on 150 epochs. And the learning rate is reduced by a factor 0.1 at the epoch 120 and 140.
